community-operators/okd-pipeline-operator/1.2.3/manifests/openshift-pipelines-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,363 @@
+              verbs: 
+                - get
+                - update
+                - create
+                - use
+                - delete
+            - apiGroups: 
+                - operator.tekton.dev
+              resources: 
+                - "*"
+              verbs: 
+                - "*"
+            - apiGroups: 
+                - tekton.dev
+              resources: 
+                - "*"
+              verbs: 
+                - "*"
+            - apiGroups: 
+                - triggers.tekton.dev
+              resources: 
+                - "*"
+              verbs: 
+                - "*"
+            - apiGroups: 
+                - console.openshift.io
+              resources: 
+                - consoleyamlsamples
+                - consoleclidownloads
+              verbs: 
+                - "*"
+          serviceAccountName: openshift-pipelines-operator
+      deployments: 
+        - name: openshift-pipelines-operator
+          spec: 
+            replicas: 1
+            selector: 
+              matchLabels: 
+                name: openshift-pipelines-operator
+            strategy: {}
+            template: 
+              metadata: 
+                labels: 
+                  name: openshift-pipelines-operator
+              spec: 
+                containers: 
+                  - command: 
+                      - openshift-pipelines-operator
+                      - "--recursive"
+                    env: 
+                      - name: WATCH_NAMESPACE
+                        valueFrom: 
+                          fieldRef: 
+                            fieldPath: "metadata.annotations['olm.targetNamespaces']"
+                      - name: POD_NAME
+                        valueFrom: 
+                          fieldRef: 
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: openshift-pipelines-operator
+                    image: "quay.io/jomeier/openshift-pipelines-operator:v1.2.3"
+                    imagePullPolicy: Always
+                    name: openshift-pipelines-operator
+                    resources: {}
+                serviceAccountName: openshift-pipelines-operator
+    strategy: deployment
+  installModes: 
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords: 
+    - tektoncd
+    - openshift
+    - build
+    - pipeline
+  links: 
+    - name: "OKD Tekton Pipeline GitHub Repository"
+      url: "https://github.com/openshift/tektoncd-pipeline"
+    - name: "OKD Pipelines Operator GitHub Repository"
+      url: "https://github.com/openshift/tektoncd-pipeline-operator"
+  maintainers: 
+    - name: "OKD working group"
+      email: jo.meier@gmx.de
+  maturity: stable
+  provider: 
+      name: OKD working group
+  version: "1.2.3"
```

Comment:
I'd like to keep this on par with the OCP operator version: https://github.com/openshift/tektoncd-pipeline-operator/blob/master/bundle/manifests/openshift-pipelines-operator.clusterserviceversion.yaml#L425

Reply:
I built the operator from the v1.2.x branch, because the "official" OpenShift Pipelines Operator has the version v1.2.3. 

The CSV in the repo was not updated since a while. The version and also the description and other fields in this CSV from the v1.2.x branch in the repo differ from the one that is installed with the official OpenShift Pipelines Operator in v1.2.3.

I'm not sure where to get this information from. The versioning of the operators is not always easy to understand.

Reply:
Maybe one of the devs of the Pipeline Operator could clarify where to find the current CSV.

Reply:
> @jomeier sorry to be such a nitpicker here,
> could you rename this PR as well as the commit message to `community: Add OKD Pipelines Operator`?
> 
> I'm not sure switching to draft is necessary here, since code-wise this lgtm 🙂. Just want somebody from the operator team to chime in (especially wrt future releases).

Done :-)